core/src/test/scala/com/lightbend/paradox/markdown/JavadocDirectiveSpec.scala

```diff
@@ -46,7 +46,7 @@ class JavadocDirectiveSpec extends MarkdownBaseSpec {
 
   it should "handle method links correctly" in {
     markdown("@javadoc[File.pathSeparator](java.io.File#pathSeparator)") shouldEqual
-      html("""<p><a href="https://docs.oracle.com/javase/8/docs/api/?java/io/File.html#pathSeparator">File.pathSeparator</a></p>""")
+      html("""<p><a href="https://docs.oracle.com/javase/8/docs/api/java/io/File.html#pathSeparator">File.pathSeparator</a></p>""")
```

Comment:
(current) https://docs.oracle.com/javase/8/docs/api/?java/io/File.html#pathSeparator 
jumps to the class page,

(expected) https://docs.oracle.com/javase/8/docs/api/java/io/File.html#pathSeparator
correctly jumps to the method section of the page

Reply:
The original behaviour of linking to the javadoc with frames comes from this suggestion by @jroper : https://github.com/lightbend/paradox/pull/52#issuecomment-258751524

I am not sure if there even exists a way to link to methods when opening with frames.

Reply:
As @jonas points out, this change results in a terrible user experience. Out of the box there doesn't exist a way to link to methods when opening with frames, but we have this Javascript code that gets added to every page in the Lagom Javadocs:

https://github.com/lagom/lagom/blob/fde496bc089157e40a3a132a3ae1cf0d5f1111d4/project/Doc.scala#L63-L79

This then gets configured here:

https://github.com/lagom/lagom/blob/fde496bc089157e40a3a132a3ae1cf0d5f1111d4/project/Doc.scala#L134

That way, you can link to methods in the docs, and still have it open with frames.

Reply:
Thanks, let me look at how lagom works for this.